test/test_structured_1d.jl

```diff
@@ -20,6 +20,12 @@ isdir(outdir) && rm(outdir, recursive=true)
       linf = [3.217887726258972e-5])
   end
 
+  @trixi_testset "elixir_advection_composite.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_composite.jl"),
+      l2   = [0.0630105298116176],
+      linf = [0.5884019851242106])
+  end
```

Comment:
Could you please rename the file? `composite` does not really tell me what's going on and it does not follow the naming scheme we used for the other examples. I think `elixir_advection_shockcapturing.jl` would fit better.